l/librdkafka/build_info.json

```diff
@@ -0,0 +1,15 @@
+{
+   "maintainer": "Balavva-Mirji",
+   "package_name": "librdkafka",
+   "github_url": "https://github.com/confluentinc/librdkafka",
+   "version": "v2.5.3",
+   "default_branch": "master",
+   "build_script": "lidbrdkafka-2.5.3_ubi_9.3.sh",
+   "package_dir": "l/librdkafka",
+   "docker_build": false,
+   "validate_build_script": true,
+   "use_non_root_user": false,
+   "v2.5.*": {
+      "build_script": "librdkafka-2.5.3_ubi_9.3.sh"
```

Comment:
Needs to be same name as the ubi 9.3 script.

Reply:
Updated as per the requirement.